popup.js

```diff
@@ -1,8 +1,41 @@
+SB = {};
+
+function configProxy() {
+    chrome.storage.onChanged.addListener((changes, namespace) => {
+        for (key in changes) {
```

Comment:
`for (key of changes)`

Reply:
I took that from https://developer.chrome.com/extensions/storage

Reply:
Should it be changed?

Reply:
You are right. It seems `in` goes through a JSON object and `of` goes through an array.